packages/foreman/foreman-webpack-helper/foreman-webpack-helper.spec

```diff
@@ -0,0 +1,40 @@
+%global release 1
+%global prerelease develop
```

Comment:
We use this for nightly packages but this will probably be rather stable so you can drop this